src/server/google-cloud.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Using Google Cloud
+short-title: Google Cloud
+description: Your Dart app can use many Google Cloud services — Firebase, Google Cloud Platform, and more.
+---
+
+Dart servers can use many
+[Google Cloud products](https://cloud.google.com/products),
+often with the help of pre-packaged [Docker images][]
+that the Dart team maintains.
+
+For information about other Google APIs (including Firebase)
+that you might want to use from Dart code,
+see the [Google APIs page](/guides/google-apis).
+
+## Cloud Run
+
+You can use Cloud Run's flexible container support,
+combined with Dart's Docker images,
+to run server-side Dart code.
+
+For more information, see the Cloud Run documentation for
+[building and deploying a service in other languages][cr].
+
+## Compute Engine
+
+To run Dart code on Compute Engine,
+use Compute Engine's support for running containers,
+combined with Dart's Docker images.
+
+For more information, see the Compute Engine documentation for
+[using software containers][ce].
+
+## Kubernetes
+
+To run Dart on clusters of Compute Engine instances,
+use Google Kubernetes Engine (GKE).
+
+For more information, see the [GKE overview][].
+
+## App Engine
+
+To run Dart code on App Engine, use
+the [`appengine` package][].
+
+{{ site.alert.warn}}
```

Comment:
This alert style seems very, very noisy on this page. It really draws the eyes, even for readers that had no intention of reading down to the App Engine content. 

Do we have some note style that is a bit more subtle, and only draws attention from those who are actually reading the AppEngine section?

Reply:
I tried rewriting it using just text, no note formatting. Check it out: https://dartlang-org-staging-1.firebaseapp.com/server/google-cloud#app-engine

Reply:
Nice!